CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.2.1-dev
```

Comment:
[nit] I think this should be 2.3.0 since it adds a new feature.

Reply:
Done.